customizations/README.md

```diff
@@ -236,6 +239,12 @@ Add the following lines to you `customizations.yaml` to connect to the external
     port: "6379"
     existingSecret: "mycarto-custom-redis-secret"
     existingSecretPasswordKey: "password"
+    tls_enabled: true
```

Comment:
This shouldn't be `false` to follow the default behaviour :thinking: ?

Reply:
Default behavior it's in `values.yaml` and already set it to `false`. Production-ready should be `true`